src/qt/forms/optionsdialog.ui

```diff
@@ -173,13 +173,23 @@
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
-             <height>20</height>
+             <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
+       <item>
+        <widget class="QCheckBox" name="enableServer">
+         <property name="toolTip">
+          <string  extracomment="Tooltip text for Options window setting that enables the RPC server.">This allows you or a third party tool to communicate with the node through command line and JSON-RPC commands.</string>
+         </property>
+         <property name="text">
+          <string extracomment="An Options window setting to enable the RPC server.">Enable &amp;RPC server</string>
```

Comment:
Tested 2c420467a980487752ae2b0c9aa89819b027ab70 on Linux Mint 20.2 (Qt 5.12.8).

The `Alt-R` shortcut is ambiguous with this PR:
```
$ git grep -n -e '&amp;R' src/qt/forms/optionsdialog.ui
src/qt/forms/optionsdialog.ui:189:          <string extracomment="An Options window setting to enable the RPC server.">Enable &amp;RPC server</string>
src/qt/forms/optionsdialog.ui:944:          <string>&amp;Reset Options</string>
```

To solve it, suggesting the following patch:
```diff
--- a/src/qt/forms/optionsdialog.ui
+++ b/src/qt/forms/optionsdialog.ui
@@ -33,7 +33,7 @@
           <string>Automatically start %1 after logging in to the system.</string>
          </property>
          <property name="text">
-          <string>&amp;Start %1 on system login</string>
+          <string>Start %1 on system &amp;login</string>
          </property>
         </widget>
        </item>
@@ -186,7 +186,7 @@
           <string  extracomment="Tooltip text for Options window setting that enables the RPC server.">This allows you or a third party tool to communicate with the node through command line and JSON-RPC commands.</string>
          </property>
          <property name="text">
-          <string extracomment="An Options window setting to enable the RPC server.">Enable &amp;RPC server</string>
+          <string extracomment="An Options window setting to enable the RPC server.">Enable RPC &amp;server</string>
          </property>
         </widget>
        </item>
```

Reply:
Is there documentation somewhere that explains how global / local these keyboard shortcuts are? I can't test it on Mac.

Reply:
Not exactly what you ask for, but https://doc.qt.io/qt-5/qshortcut.html#activatedAmbiguously

I think the scope of shortcuts is limited by a window (not a widget) which keeps the focus.